### ISSUE TITLE
[feature] Add new resource feedrecyclebin in packaging area.

### ISSUE THREAD

**Issue by C123R** [microsoft/azure-devops-go-api#67]
Currently `azure-devops-go-api` doesn't support [Permanent deletion](https://docs.microsoft.com/en-us/rest/api/azure/devops/artifacts/feed%20recycle%20bin/permanent%20delete%20feed?view=azure-devops-rest-6.0) of feeds from recycle bin.

I just realized that `azure-devops-go-api` is using a code generator. Hence just creating a feature request. It would be great if we add support for `feedrecyclebin` resource under the `packaging` area.

- GetRecycleBinFeeds(context.Context, GetRecycleBinFeedsArgs) error
- DeleteRecycleBinFeed(context.Context, DeleteRecycleBinFeedArgs) error
- RestoreRecycleBinFeed(context.Context, RestoreRecycleBinFeedArgs) error

https://docs.microsoft.com/en-us/rest/api/azure/devops/artifacts/feed%20recycle%20bin?view=azure-devops-rest-6.0

**Comment by tedchamb** [comment]
Yes, we will need to generate a 6.0 version of the SDK to get those.

**Comment by C123R** [comment]
@tedchamb Friendly reminder, as my pull request to the `terraform-provider-azuredevops` kinda depends on this:

microsoft/terraform-provider-azuredevops#306

**Comment by aleqsss** [comment]
@tedchamb, if you're about to generate a new version of the SDK, would the desired functionality of the #90 be implemented as well?

**Comment by C123R** [comment]
microsoft/terraform-provider-azuredevops#50

**Comment by EliiseS** [comment]
@xuzhang3 Is there a timeline for this feature?

**Comment by xuzhang3** [comment]
Hi @EliiseS I don't know, I have contacted service team about  when they  will release the v6 SDK,  no response yet.
